tsec-libsodium/src/main/scala/tsec/libsodium/kdf/package.scala

```diff
@@ -0,0 +1,19 @@
+package tsec.libsodium
+
+package object kdf {
```

Comment:
why do the errors classes live in the package object? they'd be perfectly-fine top level decls.